src/controllers/userController.js

```diff
@@ -0,0 +1,66 @@
+import UserSevices from '../services/UserService';
+import EncryptPassword from '../helpers/Encryptor';
+import Token from '../helpers/token';
+import response from '../helpers/responseHandler';
+/**
+* class for userControl
+ */
+class userController {
+/**
+   * signup a user and saving user data in the database
+   * @param {Object} req The request object
+   * @param {Object} res The response object
+   * @returns {Object} A user object with selected fields
+   */
+  static async signup(req, res) {
```

Comment:
### Nitpick

- You are using `Async` but I see no `Await block of code`.

Reply:
I solved it